README.md

```diff
@@ -97,9 +97,20 @@ end
 
 Quarantine comes with built-in support for the following database types:
 - `:dynamodb`
+- `:google_sheets`
 
 To use `:dynamodb`, be sure to add `gem 'aws-sdk-dynamodb', '~> 1', group: :test` to your `Gemfile`.
 
+To use `:google_sheets`, be sure to add `gem 'google_drive', '~> 2', group: :test` to your `Gemfile`. Here's an example:
+
+```rb
+config.quarantine_database = {
+  type: :google_sheets,
+  authorization: {type: :service_account_key, file: "service_account.json"}, # also accepts `type: :config`
+  spreadsheet: {type: :by_key, "1Jb5fC6wSuIMnP85tUR5knuZ4f5fuu4nMzQF6-0l-EXAMPLE"}, # also accepts `type: :by_title` and `type: :by_url`
```

Comment:
this is not a real key right ? lol

Reply:
Nah, it's fake; I ended it in `-EXAMPLE`.